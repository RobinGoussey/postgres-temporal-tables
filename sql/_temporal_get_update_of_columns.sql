CREATE EXTENSION IF NOT EXISTS psql_temporal;

--should return empty string
select _temporal_get_update_of_columns('{}');
--should return OF test, name
select _temporal_get_update_of_columns('{test,name}');

--should return OF name
select _temporal_get_update_of_columns('{name}');