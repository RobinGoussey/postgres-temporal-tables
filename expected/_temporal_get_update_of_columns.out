CREATE EXTENSION IF NOT EXISTS psql_temporal;
NOTICE:  extension "psql_temporal" already exists, skipping
--should return empty string
select _temporal_get_update_of_columns('{}');
 _temporal_get_update_of_columns 
---------------------------------
 
(1 row)

--should return OF test, name
select _temporal_get_update_of_columns('{test,name}');
 _temporal_get_update_of_columns 
---------------------------------
 OF test,name
(1 row)

--should return OF name
select _temporal_get_update_of_columns('{name}');
 _temporal_get_update_of_columns 
---------------------------------
 OF name
(1 row)

